gateway/1.4/modules/ROOT/pages/flex-conn-manage-public-api.adoc

```diff
@@ -58,7 +51,7 @@ For example, the version collected from the API Gateway HTTP basic authenticatio
 
 ** *Organization ID* for the organization where you want to run Flex Gateway
 +
-For more information, see xref:access-management::organization.adoc#find-your-organization-id[Find Your Organization ID].
+For more information, see xref:access-management::organization.adoc#find-your-organization-id[Find your Organization ID].
```

Comment:
```suggestion
For more information, see xref:access-management::organization.adoc#find-your-organization-id[Find Your Organization ID].
```

Reply:
Since you are referring directly to a topic here, you would initial cap all of the words.